templates/expo-template-bare-minimum/package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "expo": "~54.0.0-preview.1",
     "expo-status-bar": "~3.0.1",
-    "react": "19.1.0",
+    "react": "~19.1.0",
```

Comment:
this should be exact, to align with react-native: https://github.com/facebook/react-native/blob/7404fb620bb609dddd788bef16fdb9390ef2478b/package.json#L101

Reply:
There's a comment on the reasoning in the last PR. Basically patch versions should be fine in theory and it'd be nice to allow for more leeway in monorepos.

I don't think we're currently bumping our dependencies diligently, so at least, on some important ones, it'd be nice to avoid users accidentally causing themselves more conflicts and have the range be as permissive as possible (in most cases that's patch versions only though) 😅

Reply:
For example, say you're in a monorepo and also have react-dom. There's a patch to react-dom, the lockstep versions bump. You'll now have to disable the install checks or adjust them to also bump `react`, add an invalid resolution, or use force-resolution: https://app.unpkg.com/react-dom@18.3.1/files/package.json#L24